org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4whole/SimpleGUI.java

```diff
@@ -764,7 +765,9 @@ private Runner doNew() {
 	private Runner doOpen() {
 		if (wrap)
 			return wrapMe();
-		File file = OurDialog.askFile(true, null, ".als", ".als files");
+		File file = OurDialog.askFile(true, null, new String[] {
+				".als", ".md"
+		}, "Alloy (.als) or Markdown files");
```

Comment:
For consistency, change to either "Alloy or Markdown files" or "Alloy (.als) or Markdown (.md) files"